javascript/builtins/PluralRules.json

```diff
@@ -50,6 +50,58 @@
           "deprecated": false
         }
       },
+      "PluralRules": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PluralRules/resolvedOptions",
```

Comment:
The link should probably be https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules/resolvedOptions ?

Reply:
@Elchi3, I think @saschanaz is right, but can you confirm?

Reply:
Yes, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/PluralRules is correct as this is an ECMA API and not a Web API.